package.json

```diff
@@ -22,6 +22,7 @@
     "depcheck": "1.4.7",
     "eslint": "8.55.0",
     "eslint-plugin-import": "2.29.0",
+    "eslint-plugin-lit": "^1.10.1",
```

Comment:
I've been pinning all deps (no caret) and then leaning on Renovate to keep them up to date. I think that is the recommended approach for "apps" rather than "libraries". But open to discussion.

Reply:
Yeah - that's fair. Will change.